frontend/src/pages/CourseSelector/courseDescription/Collapsible.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Typography } from "antd";
+const { Title } = Typography;
+
+export const CollapsibleHeader = ({ isCollapsed, setIsCollapsed, text }) => {
+
+  const toggleCollapse = (e) => {
+    setIsCollapsed(!isCollapsed);
+  }
+
+  return (
+    <div className="collapsible-header" onClick={toggleCollapse}>
+      <Title level={3} className="text">
+        {text}
+      </Title>
+      <CollapsibleButton isCollapsed={isCollapsed} setIsCollapsed={setIsCollapsed} /> 
+    </div>
+    )
+}
+
+
+const CollapsibleButton = ({ isCollapsed, setIsCollapsed }) => {
+  const rotation = isCollapsed ? 0 : -90;
+  
+  const classes = (isCollapsed) ? "collapsible-button collapsible-button-collapsed" : "collapsible-button";
+  
+  return (
+    <div className={classes}>
+      <svg
```

Comment:
For the icon, would it better to use [antd's native icons](https://ant.design/components/icon/) instead? Since it might closely resememble the antd theme. Just a suggestion though!

Reply:
I'll probably end up going with one of the following then. 
I'm considering the `RightOutlined` due to @joooooooooooooooooooooooooooooooooooosh 's suggestion. lmk if you have a strong preference towards one or the other.
<img width="222" alt="image" src="https://user-images.githubusercontent.com/93496985/165708236-04c36cdc-ef6e-455d-bf4f-a325beddaeb8.png">

Reply:
Using RightOutlined seems a bit weird when the sidebar uses up/down arrows for collapsible containers

Reply:
Happy with the `RightOutlined` icon :D